src/tree/transform.js

```diff
@@ -72,6 +73,7 @@ export function setZooming({ objectData, setTransform, transformState, selection
   const zoomed = () => {
     setTransform({ zoom: event.transform.k / scaleFactor, x: event.transform.x, y: event.transform.y });
     bubbleEvent();
+    closeTooltip(tooltip);
```

Comment:
Is this to close when flipping between landscape/portrait?

Reply:
nah it's to close it when swiping. The touchmove listener didn't work from what I could tell.